_posts/2024-11-26-new-sponsors.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: Vox Pupuli Changelog - new sponsors
+date: 2024-11-26
+github_username: bastelfreak
+category: changelog
+---
+
+Vox Pupuli has [two new sponsors](https://voxpupuli.org/sponsoring/). CERN is sponsoring us a CI runner for unit tests, GEANT made a onetime 250$ donation to us.
```

Comment:
Is this better?
```suggestion
Vox Pupuli has [two new sponsors](https://voxpupuli.org/sponsoring/). CERN is sponsoring a CI runner for unit tests and GEANT made a onetime 250$ donation to us.
```

I also wonder if the CI runner is only for unit tests. Don't we also run acceptance tests on them?

Reply:
I removed the `unit`